### PR TITLE
Update migrating-to-v8.md

### PR DESCRIPTION
**Pull request by natanguitz**
Add warning notes

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for adding this information @natanguitz ! 🙌 

I do have one concern though - you've moved the warning up to the section about upgrading to the latest version of Umbraco 7. And therefore also **before** anything is migrated to Umbraco 8 🤔

As I understood the case here, it can be solved by one of two ways:
1. Making sure that the new Umbraco 8 site uses the same `enableLegacyEncoding` value as the Umbraco 7 site or
2. Disabling the value on the Umbraco 7 site and then run a password-recover

And I believe, if you do (1) then you will not need to do (2) 🤔 
Does that sound correct to you?

**Comment by natanguitz** [comment]
I just tried to move it in the order i think it can be better but of course I can be wrong. 

1. In the case that just copy the settings works,  yes, it didnt for me.

**Comment by sofietoft** [comment]
Right. That's a good point 🤔 

Hm. I still think that since it's currently placed **before** an Umbraco 8 site is created... Perhaps move it down here: 
image

Then there's still time to set the setting **before** anything is migrated.
What do you think about that instead?

**Comment by sofietoft** [comment]
Did you see my message here @natanguitz ? 😄 
And also, I noticed Sebastiaan adding some more comments on the Issue: https://github.com/umbraco/UmbracoDocs/issues/2747

**Comment by natanguitz** [comment]
Hello @sofietoft 

I didn't miss it, I just forgot to answer :) 

I made the change you suggested.

File: Getting-Started/Setup/Upgrading/migrating-to-v8.md [hunk 1]
```diff
@@ -84,6 +85,12 @@ First thing to do is spin up a fresh new Umbraco 8.1+ site. Make sure it all wor
 
 ![Fresh 8.1 site](images/fresh-8_1-site.png)
 
+:::warning
+If you have customized your `UsersMembershipProvider` on your Umbraco 7 site you will need to copy that over to the 8.1 web.config as well. Additionally you will need to update the `type` attribute to be `type="Umbraco.Web.Security.Providers.UsersMembershipProvider, Umbraco.Web"`.
+This also includes the attribute `useLegacyEncoding` value. In some cases you need to go back to your **Umbraco 7.14 site** , disable it and run a password restore after that in order to login succeed in your **Umbraco 8.1 site**.
```
Comment (sofietoft) [inline Getting-Started/Setup/Upgrading/migrating-to-v8.md:90]:
```suggestion

This also includes the attribute `useLegacyEncoding` value. Make sure that this setting is copied into your new Umbraco 8 site, as it is needed in order for you to be able to log in.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for moving the note @natanguitz !

I've updated the notes about the `useLegacyEncoding` setting. Instead of adding steps to resolve once it has failed, I think it makes sense to add preventive information - that way people will not have to fix it, as they will not run into it in the first place.

I hope it makes sense! Happy to answer any questions this might evoke.

**Comment by sofietoft** [comment]
Thanks for adding the suggestions @natanguitz !
And once again, thanks a lot for helping add this to the Docs - much appreciated 😄 

I say it's time we merge this in.
And I see that this is the first time you're contributing to the Umbraco Docs - that's awesome!!
Do you have an account on [Our](https://our.umbraco.com/)? If you do, I can grant you a C-Trib badge for your efforts - let me know!

**Comment by natanguitz** [comment]
Yay!   😄 yes  here => https://our.umbraco.com/members/natanguitz/   Thank you !!

**Pull request merged**